src/app/pages/map-view/sidebar-section/sidebar-section.component.ts

```diff
@@ -0,0 +1,250 @@
+import { Component, ContentChild, EventEmitter, Input, OnInit, OnChanges, Output, ViewChild } from '@angular/core';
```

Comment:
First time it was easy to notice, but this file has 4 spaces per indent level, others have 2. Either is fine, just try to be consistent with whatever you choose -- looks like you've got an editorconfig set to 2 spaces. This is also the general Angular convention.